web/src/lib/utils/file-uploader.ts

```diff
@@ -102,23 +102,6 @@ async function fileUploader(
 		// failed uploads.
 		formData.append('assetData', new File([asset], asset.name, { type: mimeType }));
 
-		// Check if asset upload on server before performing upload
-		const { data, status } = await api.assetApi.checkDuplicateAsset(
-			{
-				deviceAssetId: String(deviceAssetId),
-				deviceId: 'WEB'
-			},
-			sharedKey
-		);
-
-		if (status === 200 && data.isExist && data.id) {
-			if (albumId) {
-				await addAssetsToAlbum(albumId, [data.id], sharedKey);
-			}
-
-			return data.id;
-		}
-
```

Comment:
The web no longer does deduping, which I think is fine, since anything besides a hash is less than ideal and hashing in the UI is impractical. However, you've also removed some `addAssetsToAlbum` code. Did you uploading assets directly to an album? I'm assuming this needs to be added back.

Reply:
Just below this, the file gets uploaded and `addAssetsToAlbum` is called.  So I assume uploading to albums will still work

Reply:
I think Michel is right

Reply:
Ah ok. This was just another code branch where it would skip the upload and still add to album. No applicable anymore :+1:.